api/src/chat/services/bot.service.spec.ts

```diff
@@ -197,7 +197,7 @@ describe('BlockService', () => {
       foreign_id: 'foreign-id-web-1',
     });
 
-    event.setSender(webSubscriber);
+    event.setSender(webSubscriber!);
```

Comment:
```suggestion
    event.setSender(webSubscriber);
```
Let's use a single time at the begining.